dense-numericals-src/basic-math/arg-minimum.lisp

```diff
@@ -129,20 +129,16 @@
                                           ((keep-dims null)))
     real
```

Comment:
I think you can also change the return type of `nu:arg-maximum` from `real` to `(signed-byte 64)`. That looks compatible with the C function declarations in BMAS.

In a later update, all the C functions need to be written with platform-independent data types and not with the `long` and `int` types that they currently use. But that's separate.

Reply:
I made this change and now there's a compile error
```
The value
      64
    is not of type
      (OR CONS SYMBOL)
    when binding PELTADOT/TYPES-CORE/IMPL::TYPE-SPECIFIER
```

Is this a bug in peltadot?

Reply:
Thanks for reporting! Fixed in https://gitlab.com/digikar/peltadot/-/commit/d927039cbd9a32bfd10a3503e07455b878d7bf01